NON_CONTAINERIZED_DEV_README.md

```diff
@@ -1,49 +1,97 @@
-# Developing on SCP without a Docker container
+# Developing on SCP natively, without a Docker container
 
-Developing on SCP without a Docker container, while less robust, opens up some faster development paradigms, including 
-live css/js reloading, faster build times, and byebug debugging in rails.
+Developing on SCP natively, i.e. outside a Docker container, is less robust but faster.  In native mode, full pages load are 4-7x faster, tests run faster, and rich [byebug debugging](https://github.com/deivid-rodriguez/byebug/blob/master/GUIDE.md) in Rails is possible.
 
 ## SETUP
+Commands below assume your CPU is Apple Silicon / M1, not Intel.
+1. Append `CFLAGS="-Wno-error=implicit-function-declaration"` to your `~/.zshrc` (if using Z shell macOS default in Terminal, or `~/.bash_profile` if Bash), then run `source ~/.zschrc`
+2. Run `rbenv -v`.  If it is not found, `brew install rbenv`, then `rbenv init` to set up rbenv in your shell, then close terminal and reopen it.
+3. Run `ruby -ropenssl -e 'puts OpenSSL::OPENSSL_VERSION'`.  
+  - If it outputs contains "OpenSSL 1.1.1", go to Step 4.  
+  - Else, if output contains something like "OpenSSL 3.0.7", install the needed version of OpenSSL, and install Ruby so that it compiles with the needed version of OpenSSL:
+     - Run `brew install openssl@1.1`
+     - Run `rbenv uninstall 3.1.3`
+     - Run `LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib" CPPFLAGS="-I/opt/homebrew/opt/openssl@1.1/include" CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix openssl@1.1)" RUBY_CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix openssl@1.1)" rbenv install 3.1.3`
+     - Run `ruby -v` to ensure Ruby 3.1.3 is installed
+4. Run `ruby -v` to ensure Ruby 3.1.3 is installed on your local machine.  If not, run `rbenv install 3.1.3` (current [as of 2023-01-26](https://github.com/broadinstitute/single_cell_portal_core/pull/1713)) then `rbenv global 3.1.3`.
+5. Run `bundler -v` to ensure Bundler is installed.  If not, `gem install bundler`.
+6. Run `node -v` to ensure Node is installed. If not, install via https://nodejs.org/en/download/
+7. Run `yarn -v` to ensure Yarn is installed. If not, install via `brew install yarn`
+8. `cd` to where you have the `single_cell_portal_core` Git repo checked out.
+9. Run `bundle install`
+  - That might fail with a message that contains "An error occurred while installing bson_ext"
```

Comment:
Suggestion: move the two troubleshooting options into the Troubleshooting section at the bottom of this doc